files/es/mdn/writing_guidelines/howto/markdown_in_mdn/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Bloques de contenido
+slug: MDN/Writing_guidelines/Howto/Markdown_in_MDN
+tags:
+  - MDN
+  - Meta
+translation_of: MDN/Guidelines/CSS_style_guide
+translation_of_original: MDN/Structures/Content_blocks
+original_slug: MDN/Contribute/Markdown_in_MDN
+htmlDate: 2022-08-04T19:05:08.000Z
+---
+{{MDNSidebar}}
+
+This pages lists reusable content blocks.
+
+## Grilla de tarjetas
+
+Permite tener un par de tarjetas cercanas entre sí para llamar a contenidos específicos o para llamar a una acción. La clase CSS es: `.card-grid `(L 751 - 824 in CustomCSS).
+
+- Refetencia CSS
+  - describiendo cada propiedad y concepto de CSS.
```

Comment:
While comparing this document to the original, I found out that this document doesn't seem to match up with the English content -- like, at all.  It seems this was originally for another document that's since been removed, and was inadvertently applied to this slug.  Should we just remove this as a bad translation?